source/appModules/winword.py

```diff
@@ -54,7 +54,10 @@ class WinwordWordDocument(WordDocument):
 	def _get_description(self) -> str:
 		try:
 			curView = self.WinwordWindowObject.view.Type
-			return f"{ViewType(curView).displayString} {super()._get_description()}"
+			description = super().description
+			if isinstance(super()._get_description(), str):
```

Comment:
How about
```suggestion
			if isinstance(super()._get_description(), str) and (description := description.strip()):
```

This only returns True when the stripped description has a Truthy value, stripping description in the process, so after this if block, due to the walrus operator, description is stripped as well.

Reply:
Sounds good. I'll test this approach, which I like a lot.